packages/editor/src/lib/hooks/useCanvasEvents.ts

```diff
@@ -97,9 +88,6 @@ export function useCanvasEvents() {
 
 			function onTouchStart(e: React.TouchEvent) {
 				;(e as any).isKilled = true
-				// todo: investigate whether this effects keyboard shortcuts
```

Comment:
doesn't look like this is necessary anymore...

Reply:
did you test this on iOS? seems fine on android

Reply:
yes, i've been testing on the iOS simulator. will definitely test again once it's on staging.